src/parquet/column/levels-test.cc

```diff
@@ -206,4 +206,32 @@ TEST(TestLevelEncoder, MinimumBufferSize) {
   ASSERT_EQ(kNumToEncode, encode_count);
 }
 
+TEST(TestLevelEncoder, MinimumBufferSize2) {
+  // PARQUET-708
+  // Test the worst case for bit_width=2 consisting of
+  // LiteralRun(size=8)
+  // RepeatedRun(size=8)
+  // LiteralRun(size=8)
+  // ...
+  const int kNumToEncode = 1024;
+
+  std::vector<int16_t> levels;
+  for (int i = 0; i < kNumToEncode; ++i) {
+    if ((i % 16) < 7) {
```

Comment:
No, this is `< 7` to force a literal run. This loop generates the sequence of literal and repeated runs as mentioned in the comment above.